modules/streams-core/src/main/scala/com.snowplowanalytics.snowplow/streams/Sink.scala

```diff
@@ -23,11 +23,21 @@ trait Sink[F[_]] {
   def sinkSimple(batch: ListOfList[Array[Byte]]): F[Unit] =
     sink(batch.mapUnordered(Sinkable(_, None, Map.empty)))
 
-}
-
-object Sink {
+  /**
+   * Ping the external resource to check it is healthy and ready to receive events
+   *
+   * The sink is expected to actively ping the external resource, e.g. by sending a HTTP request.
```

Comment:
```suggestion
   * Ping the external resource to check it is healthy and ready to receive events, e.g. by sending a HTTP request.
```
I think that `actively` is misleading, when I first read this I thought that the sink would ping the resource until successful, but no there is just one request